pkg/ioutils/copy.go

```diff
@@ -0,0 +1,21 @@
+package ioutils
```

Comment:
I don't have a strong opinion about it. It seemed like a good idea because that's where we have all the other code which uses pools (or at least we did the last time I checked).